app/src/main/java/app/revanced/manager/ui/viewmodel/SelectedAppInfoViewModel.kt

```diff
@@ -155,7 +155,7 @@ class SelectedAppInfoViewModel(
 
             val previous = selectionRepository.getSelection(packageName)
             if (previous.values.sumOf { it.size } == 0) return@launch
-            selection.value = SelectionState.Customized(previous)
+            selectionState = SelectionState.Customized(previous)
```

Comment:
Can you do this for the options as well? The code is similar so it might be experiencing the same issue.